### ISSUE TITLE
Typo in the documentation

### ISSUE THREAD

**Issue by MrGretty** [arackaf/customize-cra#38]
readme.md file has a typo at  `legacyDecorators` word.

If you want use `@babel/plugin-proposal-decorators` with EsLint, you can enable `useEslintRc`, described below, with the follow configuration in your `.eslintrc` or `package.json`:

```json
{
  "extends": "react-app",
  "parserOptions": {
    "ecmaFeatures": {
      "legacyDecotators": true // this is an error
    }
  }
}
```

You need to use 
```json
{
  "extends": "react-app",
  "parserOptions": {
    "ecmaFeatures": {
      "legacyDecorators": true
    }
  }
}
```

**Comment by dalcib** [comment]
It was fixed in f61b8524671756c17abb27c762e316572b40f06e.

**Comment by arackaf** [comment]
Sweet - glad it was fixed. Thanks!
